### PR TITLE
chore: remove @gibfahn from calendar maintainers

### PR DESCRIPTION
**Pull request by bnb**
Due to inactivity (ref https://github.com/nodejs/admin/pull/594#discussion_r598060212)

### CODE REVIEW

**Review by gireeshpunathil — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -80,7 +80,6 @@ This list should be reviewed and pruned annually (at minimum). The calendar has
 <!-- sorted by GitHub handle -->
 - [@bethgriggs](https://github.com/bethgriggs) - **Beth Griggs**
 - [@bnb](https://github.com/bnb) - **Tierney Cyren**
-- [@gibfahn](https://github.com/gibfahn) - **Gibson Fahnestock**
```
Comment (nschonni) [inline README.md:83]: Move to Emeriti section?

Comment (bnb) [reply README.md:83]: You're correct, done.

**Comment by Trott** [comment]
/ping @gibfahn in case they're using this access and we just don't know about it (but in the absence of hearing anything in, say, 72 hours, I think we can go ahead and remove access--it's easy enough to re-add someone if they are removed in error).

**Review by Trott — APPROVED** [review]

**Review by gibfahn — APPROVED** [review]

**Comment by gibfahn** [comment]
:shipit: , thanks for cleaning up!

**Review by joesepi — APPROVED** [review]

**Pull request merged**